packages/plugin-evm/src/actions/swap.ts

```diff
@@ -86,7 +86,7 @@ export class SwapAction {
 export const swapAction = {
   name: 'swap',
   description: 'Swap tokens on the same chain',
-  handler: async (runtime: IAgentRuntime, message: Memory, state: State, options: any, callback?: any) => {
+  handler: async (runtime: IAgentRuntime, message: Memory, state: State, response: Memory, callback?: any) => {
```

Comment:
options is removed but still defined below `action.swap(options)` any clue what the plan was here? @shakkernerd

Reply:
We can pass it through state or add options to handlers

Reply:
@lalalune it seems that the type changes of `Handler` and rename to `ActionHandler` will break all actions.

Is that right @shakkernerd?